### ISSUE TITLE
Portal UI updates

### ISSUE THREAD

**Issue by christinayung** [cancogen-virus-seq/portal#52]
- [x]  Remove "As of April 27, 2021" from the landing page. This should ideally be auto updated with the latest submission date.  For now, remove.

![image.png](https://images.zenhubusercontent.com/5bbe9c22307bc60241e3aa93/104d8370-c268-4e66-b5b8-62d9a3f72cf6)

- [x]  Expose the `specimen collector sample ID` in the table column by default; this should be the first column in the table by default.

**Comment by caravinci** [comment]
Added the column as requested, and removed the `Release Date` from the homepage.
As it was simple enough, I've implemented simple logic to display a date in the original format, if one is provided in the helm values ([`NEXT_PUBLIC_RELEASE_DATE`](https://github.com/cancogen-virus-seq/infra/blob/main/helm/portal/dev/values.yaml#L15)).

This can be changed to read from an endpoint or any other option, but for now the value can be manually changed without code changes.
